src/parser/paladin/protection/modules/features/Checklist/Component.js

```diff
@@ -39,8 +39,13 @@ class ProtectionPaladinChecklist extends React.PureComponent{
           <AbilityRequirement spell={SPELLS.SHIELD_OF_THE_RIGHTEOUS.id} />
           <AbilityRequirement spell={SPELLS.AVENGERS_SHIELD.id} />
           <AbilityRequirement spell={SPELLS.JUDGMENT_CAST_PROTECTION.id} />
-          {combatant.hasTalent(SPELLS.BLESSED_HAMMER_TALENT.id) && <AbilityRequirement spell={SPELLS.BLESSED_HAMMER_TALENT.id} />}
-          {!combatant.hasTalent(SPELLS.BLESSED_HAMMER_TALENT.id) && <AbilityRequirement spell={SPELLS.HAMMER_OF_THE_RIGHTEOUS.id} />}
+          <Requirement
+            name={(
```

Comment:
Parenthesis could be removed and fragment on one line; but I am not sure which style is preferred